.github/workflows/package.yml

```diff
@@ -106,6 +106,9 @@ jobs:
           alembic upgrade head
           pytest --cov=argilla --cov-report=xml
 
+      - name: Setup tmate session
+        if: always()
+        uses: mxschmitt/action-tmate@v3
```

Comment:
Shouldn't we remove this block before merge? Or you prefer to leave it there for the moment? 👍🏻